forch/faucet_state_collector.py

```diff
@@ -1114,6 +1114,8 @@ def _update_stack_links_stats(self, timestamp):
     def _update_learned_macs_metrics(self):
         for mac in self.learned_macs:
             switch, port = self._get_access_switch(mac)
+            if not (switch and port):
```

Comment:
should be an else clause of the following if statement

Reply:
Addressed.